docs/devel/api_changes.md

```diff
@@ -292,6 +292,33 @@ consumers, please send an announcement to `kubernetes-dev@googlegroups.com` befo
 the change gets in. If you are unsure, ask. Also make sure that the change gets documented in
 `CHANGELOG.md` for the next release.
 
+## Defining new types
+
+Prefer to group together related concepts into a sub-struct, rather than having lots of fields at
```

Comment:
Can we just say "consider grouping related concepts into sub-structs where that makes the API clearer" or something?